aur.py

```diff
@@ -79,16 +95,27 @@ def main():
                 'default': True,
                 'type': 'bool',
             },
+            'update': {
+                'default': False,
+                'type': 'bool',
+            },
+            'auronly': {
+                'default': True,
+                'type': 'bool',
+            },
         },
+        required_one_of=[['name', 'update']],
```

Comment:
As far as I know this means that if you were to pass both, `name` will simply be ignored, but it should be more obvious that this is invalid

Reply:
I can implement update for individual name it would be cleaner, personally I dont use it but this should be allowed